setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Read in config file and assign variables
+outputs=`python read_config.py`
+vars=$(echo $outputs | tr "(',)" "\n")
+vararray=($vars)
+
+subscription=${vararray[0]}
+res_grp_name=${vararray[1]}
+location=${vararray[2]}
+cluster_name=${vararray[3]}
+node_count=${vararray[4]}
+vm_size=${vararray[5]}
+
+# Install Azure-CLI
+curl -L https://aka.ms/InstallAzureCli
```

Comment:
I think it would be worth considering wrapping this in a test to see if 'az' is already available on the system, with a successful check skipping the step, and a failure resulting in asking the user if they want to install via the OS package manager, or by a generic script. Perhaps with some rudimentary checks to see whether the user has an apt/yum/zypper-based packaging system and offering those options?